website/source/docs/connect/index.html.md

```diff
@@ -124,15 +124,12 @@ a long period of inactivity (3 days by default), the cache will empty itself.
 
 ## Multi-Datacenter
 
-Connect currently only works for service-to-service connections within a
+The OSS version of Connect only works for service-to-service connections within a
```

Comment:
This wording is strange to me since Connect isn't the product, it feels more natural to reverse this paragraph a bit:

```suggestion
Using Connect for service-to-service communications across multiple datacenters requires 
Consul Enterprise. Open Source Consul only works for service-to-service connections within
a single datacenter.
```